triton-client/src/components/App.tsx

```diff
@@ -1,33 +1,22 @@
 import { useEffect } from "react"
 import { BrowserRouter, Navigate, Route, Routes } from "react-router-dom"
-import { selectIsLoggedIn, selectLoggedInUser } from "../store/auth"
-import { useAppDispatch, useAppSelector } from "../store/hooks"
-import { selectProjects, selectProjectsLoading } from "../store/projects"
+import { selectIsLoggedIn, selectLoggedInUser } from "@store/auth"
+import { useAppDispatch, useAppSelector } from "@store/hooks"
+import { selectProjects, selectProjectsLoading } from "@store/projects"
 
 import LandingPage from "./Landing"
-import ProjectCardList from "./ProjectCardList"
+import { ProjectCardList, MGCHeader } from "./"
```

Comment:
Can we use `@components` for this import?

Reply:
done for all the other components path, im thinking of adding for something at root (src) level